stable_baselines3/common/buffers.py

```diff
@@ -263,19 +263,19 @@ def add(
         action = action.reshape((self.n_envs, self.action_dim))
 
         # Copy to avoid modification by reference
-        self.observations[self.pos] = np.array(obs)
+        self.observations[self.pos] = np.asarray(obs).copy()
```

Comment:
what is the difference between that and simply `np.array()`?

I also need to check if it's needed at all (in the sense if side effects are possible)